integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/sdx/SdxInternalTestDto.java

```diff
@@ -423,7 +423,7 @@ public SdxRepairRequest getSdxRepairRequest() {
 
     @Override
     public Clue investigate() {
-        if (getResponse() == null || getResponse().getCrn() == null) {
+        if (getResponse() == null || getResponse().getCrn() == null || getResponse().getStackV4Response() == null) {
```

Comment:
If we can have a crn in the response but not stackresponse, it means that the response could have other useful info in it. So I think a better solution would be to handle the possible NPE in the hasSpotTermination assignment, something like this:
`boolean hasSpotTermination = Optional.ofNullable(getResponse().getStackV4Response())
                .map(stack -> stack.getInstanceGroups())
                .stream()
                .flatMap(ig -> ig.getMetadata().stream())
                .anyMatch(metadata -> InstanceStatus.DELETED_BY_PROVIDER == metadata.getInstanceStatus());`
This way we could add the response to the report without an NPE. Also it is possible that other Investigables have this issue, please check them too.

Reply:
so you mean audit info still useful, whether stackresponse is null?

Reply:
I think it can be as it already had a crn, because without it could not have caused an NPE.

Reply:
other testdtos had the potential issue too, it was a good idea to check them. Thanks